Sources/Core/Filters/FilterViewController.swift

```diff
@@ -83,6 +82,6 @@ class FilterViewController: UIViewController, FilterBottomButtonViewDelegate {
     // MARK: - FilterBottomButtonViewDelegate
 
     func filterBottomButtonView(_ filterBottomButtonView: FilterBottomButtonView, didTapButton button: UIButton) {
-        delegate?.filterViewControllerDidSelectApply(self)
+        navigationController?.popToRootViewController(animated: true)
```

Comment:
I don't think it's a good practice to access `navigationController` from view controller because then we assume that it was pushed into navigation stack. Nothing will happen if we call `popToRootViewController` from a presented view controller for instance. That's why I'd better call this delegate method in order to keep view controller unaware of navigation.

Reply:
👍